modules/gdscript/gdscript_function.cpp

```diff
@@ -1549,7 +1549,9 @@ Variant GDScriptFunction::call(GDScriptInstance *p_instance, const Variant **p_a
 		// function, file, line, error, explanation
 		String err_file;
 		if (p_instance)
-			err_file = p_instance->script->path;
+			if(p_instance->script->path != "") {
```

Comment:
Style issue, there should be a space after `if`.

Reply:
Fixed, and I am curious about something. Why we not use a static reference to check a string is empty or null instead of creating new string everytime like `""`?

Reply:
You can use `!path.empty()` if you want. It would be interesting to benchmark what difference it makes, if any.